packages/x-components/src/x-modules/tagging/store/actions/track-tagging.action.ts

```diff
@@ -0,0 +1,45 @@
+import { TaggingInfo } from '../../../../../../search-types';
+import { XPlugin } from '../../../../plugins/x-plugin';
+import { DefaultSessionService } from '../../service';
+import { TaggingXStoreModule } from '../types';
+
+/**
+ * Default implementation for the {@link TaggingActions.trackTagging}.
+ *
+ * @param context - The {@link https://vuex.vuejs.org/guide/actions.html | context} of the actions,
+ * provided by Vuex.
+ * @param taggingInfo - The taggingInfo.
+ *
+ * @public
+ */
+export const trackTagging: TaggingXStoreModule['actions']['trackTagging'] = (
+  context,
+  taggingInfo
+) => {
+  const consent = context.state.consent;
+  const tagging = Array.isArray(taggingInfo) ? taggingInfo : [taggingInfo];
+
+  tagging.forEach(({ url, params }: TaggingInfo) => {
+    const sessionId = getSessionId(consent);
+    return XPlugin.adapter.track({
```

Comment:
As far as I know we're not waiting for this response, we just track and move on.

Reply:
I added because the IDE was complaining:

![image](https://user-images.githubusercontent.com/3502075/142832098-28d423b6-a653-4437-98d0-d8d8218f7c3e.png)

Reply:
I don't think we need to wait for any tracking call, as there's no response returned. You should be able to remove the `return` and don't listen to the IDE here, as it is just a reminder.